lib/main.js

```diff
@@ -189,6 +200,21 @@ export default {
     });
   },
 
+  async getProgram(Linter, configurationPath) {
+    let program;
+    const tsconfigPath = path.resolve(path.dirname(configurationPath), 'tsconfig.json');
+    try {
+      const stats = await stat(tsconfigPath);
+      const isFile = stats.isFile();
+      if (isFile) {
+        program = Linter.createProgram('tsconfig.json', path.dirname(configurationPath));
```

Comment:
Fixed in https://github.com/AtomLinter/linter-tslint/pull/157/commits/8c8f7f70e20332d44bb5f72bafe27e635298dd5b